pkg/cloudprovider/cloudprovider.go

```diff
@@ -53,10 +118,15 @@ func (d *defaultProvider) GetZone(node *v1.Node) (string, error) {
 	return node.Labels[failureDomainZoneKey], nil
 }
 
-func init() {
+func (d *defaultProvider) IsK8sServiceProvider() bool {
+	return false
+}
+
+func initRegistry() {
 	providerRegistryLock.Lock()
 	defer providerRegistryLock.Unlock()
 
 	providerRegistry = make(map[string]Ops)
 	providerRegistry[azureName] = &azure{}
```

Comment:
[nit] it's strange having `AWSProviderName` be public but `azureName` be private.

Reply:
will import the names from cloudops vendor soon.